hadoop-hdds/docs/themes/ozonedoc/static/swagger-resources/recon-api.yaml

```diff
@@ -0,0 +1,1584 @@
+openapi: 3.0.0
+info:
+  title: Ozone Recon REST API
+  license:
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+    name: Apache 2.0 License
+servers:
+  - url: /api/v1/
+tags:
+  - name: Containers and Keys
+    description: APIs to fetch information about the available containers and keys. **Admin Only**
+  - name: Blocks Metadata
+    description: APIs to fetch metadata for the blocks available. **Admin Only**
+  - name: Namespace Metadata
+    description: APIs to fetch metadata for the namespace. **Admin Only**
+  - name: Cluster State
+    description: APIs to fetch data about the cluster state
+  - name: Datanodes
+    description: APIs to fetch data about the Datanodes
+  - name: Pipelines
+    description: APIs to fetch data about the Pipelines
+  - name: Tasks
+    description: APIs to fetch data about status of Recon Tasks
+  - name: Utilization
+    description: APIs to fetch data about space utilization
+  - name: Metrics
+    description: APIs to fetch data about various metrics from Prometheus
+    externalDocs:
+      description: Prometheus API docs
+      url: https://prometheus.io/docs/prometheus/latest/querying/api/
+paths:
+  /containers:
+    get:
+      tags:
+        - Containers and Keys
+      summary: Get all Container Metadata information
+      operationId: getContainerInfo
```

Comment:
How this operationId gets generated ?

Reply:
So operationId will not be displayed in the UI, it is a unique identifier internally for Swagger, so that in future if we want to reference some api we can use that.


----
From [swagger docs](https://swagger.io/docs/specification/paths-and-operations/):
Some common use cases for operationId are:

* Some code generators use this value to name the corresponding methods in code.
* [Links](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#linkObject) can refer to the linked operations by operationId.

Reply:
> So operationId will not be displayed in the UI, it is a unique identifier internally for Swagger, so that in future if we want to reference some api we can use that.
> 
> From [swagger docs](https://swagger.io/docs/specification/paths-and-operations/): Some common use cases for operationId are:
> 
> * Some code generators use this value to name the corresponding methods in code.
> * [Links](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#linkObject) can refer to the linked operations by operationId.

Can we change it to reflect method name ?

Reply:
Sure, we can do that

Reply:
Addressed in the latest commit